common/src/main/java/io/netty/util/concurrent/EventExecutorGroup.java

```diff
@@ -116,4 +116,17 @@
 
     @Override
     ScheduledFuture<?> scheduleWithFixedDelay(Runnable command, long initialDelay, long delay, TimeUnit unit);
+    
+    /**
+     * Set the reject task handler that will be called if the execution of a task fails. 
+     * See https://github.com/netty/netty/issues/3449
```

Comment:
Remove the issue from the javadoc... maybe add it as a comment but not as part of the javadoc.